ocfl-java-core/src/main/java/io/ocfl/core/OcflRepositoryBuilder.java

```diff
@@ -411,7 +411,7 @@ public MutableOcflRepository buildMutable() {
         return buildInternal(DefaultMutableOcflRepository.class);
     }
 
-    private <T extends OcflRepository> T buildInternal(Class<T> clazz) {
+    protected <T extends OcflRepository> T buildInternal(Class<T> clazz) {
```

Comment:
Why does this need to be made `protected`? If you're subclassing `OcflRepositoryBuilder`, wouldn't you just override `build()`?

Reply:
True, I will fix this.